src/node_metadata.rs

```diff
@@ -265,12 +320,14 @@ impl<K: Encode, V: Decode + Clone> StorageMap<K, V> {
         bytes.extend(&sp_core::twox_128(&self.storage_prefix)[..]);
         let encoded_key = key.encode();
         let hash = match self.hasher {
+            StorageHasher::Identity => encoded_key.to_vec(),
             StorageHasher::Blake2_128 => sp_core::blake2_128(&encoded_key).to_vec(),
             StorageHasher::Blake2_128Concat => {
                 // copied from substrate Blake2_128Concat::hash since StorageHasher is not public
-                sp_core::blake2_128(&encoded_key)
+                let x: &[u8] = &encoded_key[..];
```

Comment:
you could as well use `encoded_key.as_slice()` , which I personally prefer

Reply:
ok. was copy-paste from upstream, but I agree